src/api/getSingleMessage.js

```diff
@@ -0,0 +1,12 @@
+import { apiGet } from './apiFetch';
+
+export default async (
+  auth,
+  messageId,
+) =>
+  apiGet(
+    auth,
+    `messages/${messageId}`,
+    {},
+    res => res,
```

Comment:
No issues with performance. The goal of this function is to extract the relevant data from the API call, which is often `res.something` while the `something` part varies, thus the need for a function.